app/src/main/java/com/kms/cura/model/MessageModel.java

```diff
@@ -84,4 +84,27 @@ public boolean deleteMessage(UserEntity userEntity, MessageEntity entity) throws
         throw new Exception(response.getError());
     }
 
+    public boolean insertMessage(MessageEntity entity) throws Exception {
+        StringBuilder builder = new StringBuilder();
+        builder.append(Settings.SERVER_URL);
+        builder.append(Settings.INSERT_MESSAGE);
+        JsonElement element = EntityToJsonConverter.convertEntityToJson(entity);
+        JsonObject object = (JsonObject) element;
+        if (entity.isSenderDoctor()){
+            object.addProperty(MessageEntity.SENT_BY_DOCTOR, true);
+        }
+        else{
+            object.addProperty(MessageEntity.SENT_BY_DOCTOR, false);
+        }
```

Comment:
use `object.addProperty(MessageEntity.SENT_BY_DOCTOR, entity.isSenderDoctor());` to avoid if else block